NEWS.md

```diff
@@ -1,5 +1,20 @@
 ## CRE development
 
+## CRE 0.0.3 (2022-02-23)
```

Comment:
Please only modify what you have added, or changed, or deleted. Do not change this subtitle.